ext/phar/tests/cache_list/files/frontcontroller17.phar.inc

```diff
@@ -6,7 +6,7 @@ echo "hi";
 ';
 $a->setStub('<?php
 try {
-Phar::webPhar("test.phar", "/index.php", null, array(), "sort");
+Phar::webPhar("test.phar", "/index.php", null, array(), function() { throw new Exception; });
```

Comment:
They were testing calls where zend_is_callable succeeds, but zend_call_function fails. There's no good way to test this anymore, so I switched to throwing an exception -- it's not quite the same, but similar.